pkg/controller/test-images-distributor/test_images_distributor_test.go

```diff
@@ -726,7 +726,7 @@ func TestTestInputImageStreamTagFilterFactory(t *testing.T) {
 			expectedResult: true,
 		},
 		{
-			name: "imagestreamtag is referenced by imagestreatag import",
+			name: "imagestreamtag is referenced by imagestreamtag import",
```

Comment:
NIT: I guess this existing test does not reflect the change in the PR.
Maybe we can trust the replace function.

Reply:
It does reflect the change in the PR, `WithImageStreamLabels` adds the labels (which now replace colon with unerscore) and verifies that that results in the filter allowing it